Server/README.md

```diff
@@ -62,3 +62,27 @@ NOTE: the above token is invalid do not try using it
  * [Upload Files](documentation/uploadFiles.md)
  * [Blobs](documentation/azureBlob.md)
  * [Containers](documentation/azureContainer.md)
+
+# Backend server-side socket documentation
+
+URL: localhost:8080
```

Comment:
url should be consistent with what's in the server

Reply:
I put that URL because that it what works in Postman